demos/sample-javascript/src/main.js

```diff
@@ -8,27 +9,54 @@ const hostUrlEl = document.getElementById("hostUrl");
 const whoAmIResponseEl = document.getElementById("whoamiResponse");
 const canisterIdEl = document.getElementById("canisterId");
 const principalEl = document.getElementById("principal");
+const delegationEl = document.getElementById("delegation");
+const expirationEl = document.getElementById("expiration");
 const iiUrlEl = document.getElementById("iiUrl");
+const maxTimeToLiveEl = document.getElementById("maxTimeToLive");
 
 let authClient;
 
 const init = async () => {
   authClient = await AuthClient.create();
-  principalEl.innerText = await authClient.getIdentity().getPrincipal();
 
-  // Redirect to the identity provider
+  const updateView = () => {
+    const identity = authClient.getIdentity();
+
+    principalEl.innerText = identity.getPrincipal();
+    if (identity instanceof DelegationIdentity) {
+      delegationEl.innerText = JSON.stringify(identity.getDelegation().toJSON());
```

Comment:
```suggestion
      delegationEl.innerText = JSON.stringify(identity.getDelegation().toJSON(), undefined, 2);
```

If you'd like it to pretty print the JSON with some spacing and newlines